diplom.py

```diff
@@ -40,44 +57,26 @@ def parse(line):
     multi_class = 'multinomial', max_iter = 100)
 print('LogisticRegression score: %f'
       % logistic.fit(train_X, train_y).score(test_X, test_y))
-Predicted_probability_distribution = logistic.predict_proba(test_X)
-#print logistic.classes_
+predicted_probability_distribution_test = logistic.predict_proba(test_X)
+predicted_probability_distribution_train = logistic.predict_proba(train_X)
+list_of_classes = logistic.classes_
```

Comment:
Inline in this case means remove the variable definition and just pass "logistic.classes_" as argument two function calls.